Formula/now-cli.rb

```diff
@@ -0,0 +1,15 @@
+class NowCli < Formula
+  desc "The command-line interface for Now"
+  homepage "https://zeit.co/now"
+  url "https://github.com/zeit/now-cli/releases/download/15.8.0/now-macos.gz"
+  sha256 "350017e2010017e84a34a459b398f3bed24f0ffd556c78fe25c03c1ac64c3521"
+
+  def install
+    bin.install "now-macos"
+    mv "#{bin}/now-macos", "#{bin}/now"
+  end
+
+  test do
+    system "#{bin}/now", "-v"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I've add a basic usage of it, please help to review.